tools/libSwiftSyntaxParser/libSwiftSyntaxParser.cpp

```diff
@@ -101,16 +128,6 @@ class CLibParseActions : public SyntaxParseActions {
     }
   }
 
-  void makeCRange(CRange &c_range, CharSourceRange range) {
```

Comment:
You don't have to remove it, you can have this method delegate to the global static function, e.g.:
```
void makeCRange(CRange &c_range, CharSourceRange range) {
  return makeCRange(c_range, range, SM, BufferID);
}
```
that way you don't have to update the callers and have to pass the extra parameters at the call sites.